Directory.Build.props

```diff
@@ -19,20 +19,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Temporalio" Version="1.1.2" />
-    <PackageReference Include="Temporalio.Extensions.DiagnosticSource" Version="1.1.2" />
-    <PackageReference Include="Temporalio.Extensions.Hosting" Version="1.1.2" />
-    <PackageReference Include="Temporalio.Extensions.OpenTelemetry" Version="1.1.2" />
+    <PackageReference Include="Temporalio" Version="1.2.0" />
+    <PackageReference Include="Temporalio.Extensions.DiagnosticSource" Version="1.2.0" />
+    <PackageReference Include="Temporalio.Extensions.Hosting" Version="1.2.0" />
+    <PackageReference Include="Temporalio.Extensions.OpenTelemetry" Version="1.2.0" />
     <!--
     Can also reference the SDK downloaded to a local directory:
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\temporal-sdk-dotnet\src\Temporalio\Temporalio.csproj" />
     -->
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.4.33" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
```

Comment:
Had to update logging here because the OpenTelemetry 1.9.0+ library has a dependency on 8.0.0+

![image](https://github.com/user-attachments/assets/ed186d2e-749e-42fb-ab74-c5df6a368aa0)